recipes/pytest-picked/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "pytest-picked" %}
+{% set version = "0.2.0" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "426a5787d0a9398f628302a09aaf71bd1cb2706b38fc78939d18c44279842749" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install . --no-deps --ignore-installed
```

Comment:
Hi @nicoddemus , could you please change this line to:
```
"{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```

This is the new default for conda to install python projects

Reply:
Didn't know that changed, thanks!